src/frontend/planner_test/tests/testdata/subquery.yaml

```diff
@@ -165,3 +165,42 @@
                 BatchScan { table: t1, columns: [], distribution: SomeShard }
           BatchExchange { order: [], dist: Single }
             BatchScan { table: t2, columns: [], distribution: SomeShard }
+- sql: |
```

Comment:
How about adding all these queries mentioned in https://github.com/risingwavelabs/risingwave/issues/5297 to here?

Reply:
Actually they look very similar, except the filter condition on `relkind`. Only keep `\d` here is okay I think. 🤔